src/App.tsx

```diff
@@ -1,42 +1,33 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { Tab } from './react-app-env';
```

Comment:
Don't change or modify this file `react-app-env.d.ts` - it's special type declarations file. Instead create folder called `types` and add new type files there.